src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -36,6 +36,9 @@ internal partial class CurlHandler : HttpMessageHandler
         private const string NoContentType = HttpKnownHeaderNames.ContentType + ":";
         private const int CurlAge = 5;
         private const int MinCurlAge = 3;
+        private const string ClientAuthenticationOID = "1.3.6.1.5.5.7.3.2";
+        private const int SecurePasswordByteLength = 52;
```

Comment:
Ah, you do have the 52 value in a named constant.  You should use this name instead of the 52 you explicitly wrote out earlier.